components/TaskCompleted.js

```diff
@@ -0,0 +1,61 @@
+import { TaskTitle } from "./Task";
+import TaskItem from "./TaskItem";
+import styled from "styled-components";
+
+export default function TaskCompleted({ task, handleToggleTask }) {
+  if (task === undefined || task === null || task.length === 0) return null;
+
+  const summeTask = task.length;
+  const amountCompletedTask = task.filter((item) => item.checked).length;
```

Comment:
These two variables are similar in content. Perhaps they should also be named similarly. E.g. `tasksAmountTotal` and `tasksAmountCompleted`?

Reply:
good variable names! thank you Kai 👍